docs/user-guide/systemrequirements.md

```diff
@@ -115,7 +125,7 @@ Configure zowe-setup-certifcates.env using the following parameters. Both are re
 
 ## Zowe CLI requirements (client)
 
-Zowe CLI is supported on platforms where Node.js 8.0 or 10 is available, including Windows, Linux, and Mac operating systems. Zowe CLI was also tested and confirmed to run on Unix System Services (USS) on z/OS.
+Zowe CLI is supported on platforms where Node.js 8.0 or 10 is available, including Windows, Linux, and Mac operating systems. 
```

Comment:
Perhaps this should be "Node.js 8.0 or higher" LTS versions (as is stated below).